server/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,7 +15,8 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('firstname');
```

Comment:
make a new migration file to avoid possibility of ruining sa deployment operations sa future if ever.